cmd/llb.go

```diff
@@ -56,9 +77,8 @@ and outputs buildkit LLB to stdout. This can be used as 'bldr pack ... | buildct
 }
 
 func init() {
-	llbCmd.Flags().StringVarP(&options.Target, "target", "t", "", "Target image to build")
+	llbCmd.Flags().StringVarP(&target, "target", "t", "", "Target image to build")
 	llbCmd.MarkFlagRequired("target") //nolint: errcheck
-	llbCmd.Flags().Var(&options.BuildPlatform, "build-platform", "Build platform")
-	llbCmd.Flags().Var(&options.TargetPlatform, "target-platform", "Target platform")
+	llbCmd.Flags().StringVar(&platform, "platform", "", "Target platform")
```

Comment:
should this have sane default, like `environment.LinuxAmd64`?

Reply:
Fixed.